docs/data-management/connecting-dwh/bigquery.md

```diff
@@ -17,6 +17,10 @@ Additionally, you will need to create a data environment for Eppo to write inter
 7. Under **Service account permissions**, add the following roles:
     - `BigQuery Job User (roles/bigquery.jobUser)`
       - Required
+    - `BigQuery User (roles/bigquery.user)`
```

Comment:
AI suggests `roles/bigquery.dataEditor` may accomplish what we need with less permissions than user